src/router/routes.ts

```diff
@@ -43,12 +43,16 @@ export const routes = [
       requiresAuth: true
     }
   },
+  //
+  // TODO: handle steps differently because they
+  //       differ depending on company type (BC, CP)
+  //
   {
     path: '/review-confirm',
     name: 'review-confirm',
     component: ReviewConfirm,
     meta: {
-      step: 2,
+      step: 5,
```

Comment:
i like the use of the meta data here!

Reply:
Thanks but again, we'll have different steps depending on business type, and how we manage those will affect the routing and knowing what step we're on. My vote is leave it for now. What do you think?

Reply:
I agree to leave it for now as well. Down the road we might want to use the entityType to define what `steps` are involved in the stepper. Possible defining each step in an enum, and depending on what entity type we can build an array of steps from said enum. Which the Stepper can then use to navigate. How this will tie into the routing, not 100% sure lol. Just a random thought!

Reply:
Separate enums might work, or an array of steps, I dunno. It might be neat to tie in to Peter's resource stuff too. I'll defer this to later!